DEVELOPMENT.md

```diff
@@ -53,6 +53,8 @@ This is done by running `yarn changeset`, which will guide you through all of th
 
 Commit the resulting changeset file with your other changes and push it up. This can happen at any time in the lifecycle of the branch.
 
+Changesets isn't smart enough to detect dependent changes and version accordingly. If you are only changing code in the `@chromatic-com/shared-e2e` package, you will need to explicitly include the `chromatic-com/playwright` and `chromatic-com/cypress` packages as packages to version-bump as well.
+
```

Comment:
@tevanoff come to think of it, do we even need to include the `@chromatic-com/shared-e2e` package as a package to version-bump, or can we always leave it out? I think we have been inconsistent in bumping it when changes are made.